docs/source/tutorial_material/tutorial.ipynb

```diff
@@ -195,7 +199,326 @@
+      "text/plain": [
+       "<Figure size 200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from eulerpi.core.plotting import sample_violin_plot\n",
+    "\n",
+    "# plot the inferred parameter distribution\n",
+    "sample_violin_plot(model)\n",
+    "# plot the inferred data distribution\n",
+    "sample_violin_plot(model, what_to_plot=\"data\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The two plots correspond to the inferred parameter distribution $\\hat{\\Phi}_{\\mathcal{Q}}$ and a KDE of the pushforward of the inferred sample, i.e. $s(\\hat{q_i})$. We will come back to the plotting function when we defined a more interesting model:\n",
```

Comment:
Maybe already give a hint/reminder, what the plot of the KDE of $s(\hat q_i)$ shows?